lib/drivers/nash.js

```diff
@@ -0,0 +1,58 @@
+const Driver = require('../models/driver');
+const Supply = require('../models/supply');
+
+/**
+ * Nash driver. Supports max
+ * supply and circulating supply
+ * for NEX token on NEO blockchain.
+ *
+ * @memberof Driver
+ * @augments Driver
+ */
+class Nash extends Driver {
+  constructor(options) {
+    super({
+      timeout: 100, // 10 requests per second
+      supports: {
+        total: false,
+        max: true,
+        circulating: true,
```

Comment:
@nickpater I wasn't sure should I define blockchains property to NEO or not.

Reply:
No this is perfect haha!